tensorflow/core/BUILD

```diff
@@ -1348,9 +1380,8 @@ tf_cuda_library(
         ":lib",
         ":lib_internal",
         ":protos_all_cc",
-        ":stream_executor",
         "//third_party/eigen3",
-    ],
+    ] + if_not_windows([":stream_executor"]),
```

Comment:
Some of these dependencies look spurious (especially for `ctc_beam_search_lib`), but fixing that is probably orthogonal to this change. (`"gpu_runtime"` seems like a strange thing for CPU-only code to depend on :)....) Fine to leave as is.